src/main/java/com/openshift/internal/restclient/capability/resources/AbstractOpenShiftBinaryCapability.java

```diff
@@ -191,6 +193,15 @@ private ProcessBuilder initProcessBuilder(String location, final OpenShiftBinary
 		return builder;
 	}
 	
+	private List<String> splitTakingCareOfSpaceInsiqueQuotes(String str){
```

Comment:
Is there a misspelling here? Should it be `splitTakingCareOfSpaceInsideQuotes' .  Can we just call it 'split' since that is what it is doing or 'normalize' or 'sanitize'

Reply:
yes, mispelling.
I would prefer to have a more meaningful than just split/normalize/sanitize